products/revenue_analytics/frontend/tiles/RevenueGrowthRateTile.tsx

```diff
@@ -19,10 +27,59 @@ const INSIGHT_PROPS: InsightLogicProps<InsightVizNode> = {
 }
 
 export const RevenueGrowthRateTile = (): JSX.Element => {
+    const { baseCurrency } = useValues(revenueEventsSettingsLogic)
+
     const { queries } = useValues(revenueAnalyticsLogic)
     const query = queries[QUERY_ID]
 
-    const context = useMemo(() => ({ insightProps: { ...INSIGHT_PROPS, query } }), [query])
+    const columns: QueryContext['columns'] = useMemo(() => {
+        return {
+            month: { title: 'Month' },
+            mrr: {
+                title: 'MRR',
+                render: ({ value }) => <MRRCell value={value as number} currency={baseCurrency} />,
+            },
+            previous_mrr: {
+                title: 'Previous MRR',
+                render: ({ value }) => <MRRCell value={value as number} currency={baseCurrency} />,
+            },
+            mrr_growth_rate: {
+                title: 'MRR Growth Rate',
+                render: ({ value, recordIndex, rowCount }) => (
+                    <MRRGrowthRateCell value={value as number} isLast={recordIndex === rowCount - 1} />
```

Comment:
In case you're curious, this is why I had to add a `rowCount` property to our table.